### PR TITLE
Changes made to the challenges in spanish.

### PR DESCRIPTION
**Pull request by theHazzard**
I've changed the challenges section of every exercise, the user must write the solution in english for the verification to be successful, so if we provide spanish examples the user will write it in spanish too, and it will fail in the verification step.

### CODE REVIEW

**Comment by theHazzard** [comment]
@a0viedo

**Comment by esalinas00** [comment]
In #84 @a0viedo  and me tried to discuss if the variables' names should be in english or only the string literals.
I vote for variables' name en english (the way it is in your PR), reasons:
1. IMO it would be weird having variables' names in english while just literals in english.
2. The other translations also have their variables' name in english. (convention)
3. Most variables' names are simple english words, ppl wouldn't need a high level of fluency in english
to undestand them.

File: problems/accessing-array-values/problem_es.md [hunk 1]
```diff
@@ -9,35 +9,35 @@ El número de índice comienza en cero y finaliza en el valor de la propiedad lo
...
 
 ```js
-	var mascotas = ['gato', 'perro', 'rata'];
+	var pets = ['cat', 'dog', 'rat'];
 
-	console.log(mascotas[0]);
+	console.log(pets[0]);
 ```
 
 El código de arriba, imprime el primer elemento del array de `mascotas` - string `gato`
```
Comment (esalinas00) [inline problems/accessing-array-values/problem_es.md]: that would be pets and cat now

Comment (theHazzard) [inline problems/accessing-array-values/problem_es.md]: thanks, I'm going to correct this right now, I've tried to touch the translations the least I could, I've forgotten to change that

**Comment by a0viedo** [comment]
LGTM, just tried the `verify` command in this PR and works perfectly. thanks @theHazzard!

**Pull request merged**